pkg/cmos/cmos.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2012-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package cmos lets you read and write to cmos registers while doing basic checks on valid register selections.
+
```

Comment:
no empty line between the package comment and package statement. stmt must immediately follow comment, or the godoc parser won't pick it up

Reply:
argh, sorry, i meant the other way.

```go
// copyright ...

// Package cmos...
package cmos
```